h/presenters/group_json.py

```diff
@@ -10,27 +10,42 @@ def __init__(self, group, links_svc=None):
         self.group = group
         self._links_svc = links_svc
 
-    def asdict(self):
-        return self._model(self.group)
+    def asdict(self, expand=[]):
+        model = self._model()
+        self._expand(model, expand)
+        self._inject_urls(model)
+        return model
+
+    def _expand(self, model, expand=[]):
+        if 'organizations' in expand:
+            org_model = {}
+            org = self.group.organization
+            if org is not None:
+                org_model = {
```

Comment:
The client will need the logo URL as well

Reply:
Yup! See the description from this PR:

> Next up:

> Adding logo
> Refactor/create an OrganizationJSONPresenter

Reply:
Right, sorry. I totally missed that.